src/lib/api.ts

```diff
@@ -1046,6 +1046,41 @@ export const searchParentContent = cache(
     },
 );
 
+/**
+ * Search content in a Site or specific SiteSpaces.
+ */
+export const searchSiteContent = cache(
+    'api.searchSiteContent',
+    async (
+        organizationId: string,
+        siteId: string,
+        query: string,
+        siteSpaceIds: string[] = [],
+        /** A cache bust param to avoid revalidating lot of cache entries by tags */
+        cacheBust?: string,
```

Comment:
It doesn't look like this is used anywhere in the code?

Reply:
It's not used, but passed; which is then computed in the cacheKey

Reply:
Sorry but I can't see where this is even passed?
https://github.com/GitbookIO/gitbook/blob/47fbf27ff2a25d1a0dbc12b26ed63f838b510b85/src/lib/api.ts#L1052-L1082

Reply:
Do you mean how it's passed down from the frontend to the API or do you mean how it acts as a cache breaking arg? The latter is here: https://github.com/GitbookIO/gitbook/blob/main/src/lib/cache/cache.ts#L214